src/session/content/message_row/media_picture.rs

```diff
@@ -0,0 +1,189 @@
+use gtk::{gdk, glib, graphene, prelude::*, subclass::prelude::*};
+
+const MIN_WIDTH: i32 = 100;
+const MIN_HEIGHT: i32 = 100;
+const MAX_HEIGHT: i32 = 400;
+
```

Comment:
Just want to point out that the scope of these could be narrower as they are only used in the `imp` module. Ignore this, if they are at that place for better readability.